lib/ansible/executor/playbook_executor.py

```diff
@@ -168,8 +168,9 @@ def run(self):
                 # send the stats callback for this playbook
                 if self._tqm is not None:
                     if C.RETRY_FILES_ENABLED:
-                        retries = list(set(self._tqm._failed_hosts.keys() + self._tqm._unreachable_hosts.keys()))
-                        retries.sort()
+                        retries = set(self._tqm._failed_hosts.keys())
+                        retries.update(self._tqm._unreachable_hosts.keys())
+                        retries = sorted(list(retries))
```

Comment:
Just discovered, don't need the intermediate list here, either.  sorted takes an iterable so ```new_list = sorted(set((5,4,3,2,1)))``` is perfectly valid.